### ISSUE TITLE
Bug? Les profs laissent incendies/portails durant leurs cours

### ISSUE THREAD

**Issue by Salvador1312** [JulienNigon/SpellcasterUniversity-Bug-Report#808]
Je ne sais pas comment classer ça, car je ne sais pas si c'est un dysfonctionnement ou une incohérence, mais si un prof fait cours et qu'un ptit malin d'élève déclenche un incendie ou ouvre un portail durant celui-ci, le prof continue de faire cours (dans le vide car les étudiants fuient la salle) et c'est un autre professeur (en pause souvent) qui vient régler l'affaire pendant que le premier fait style de bosser.

![20200208234938_1](https://user-images.githubusercontent.com/60800668/74334771-d6aa2a80-4d9a-11ea-9ca2-d6b502638fbe.jpg)

**Comment by kaori01** [comment]
Je ne sais pas si c'est un bug, mais j'ai déjà constaté ça, le prof qui vient de super loin pour réparer pendant que le prof présent dans la salle fait style j'ai rien vu, alors que ses élèves eux sont en train de hurler tout autours... :D

**Comment by Florane88** [comment]
Absolument ! Les profs sont à 250% dans leur cours et rien, absolument rien ne les déconcentrera !! XD

**Comment by Neelan22** [comment]
Oui, l'intelligence des professeurs doit être améliorée sur ça ^^

**Comment by Juliusmagnus1986** [comment]
Ah je croyais que c'était fait exprès, sans quoi, les 3/4 des portails /incendies seraient réglés quasi immédiatement, ce qui les rendrait beaucoup moins pénible et du coup presque inutiles dans le gameplay.

**Comment by kaori01** [comment]
J'ai cru que c'était fait exprès parce que c'est tellement énorme que c'en est marrant 😆

**Comment by Neelan22** [comment]
Juliusmagnus1986 ça reste un problème car le prof arrête quand même son cours et les élèves sont partis faire autre chose.
